rp2040-hal/src/pwm/mod.rs

```diff
@@ -0,0 +1,581 @@
+//! Pulse Width Modulation (PWM)
+//!
+//! To access the PWM pins you must call the 'split' method on the PWM. This will return a
```

Comment:
This documentation along with the examples needs to be updated.

Reply:
Where is the split method? What is a `_____` struct?
I think this first bit of the docs needs little more attention.